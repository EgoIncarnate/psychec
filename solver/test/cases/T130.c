void generate()
{
    struct T v;
    Chain c = v;
}
