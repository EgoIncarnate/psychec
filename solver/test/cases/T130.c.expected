typedef struct T {} T;
typedef struct T Chain;
